fhir-persistence/src/main/java/com/ibm/fhir/persistence/HistorySortOrder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.persistence;
+
+
+/**
+ * The ordering specified in the client request for whole system history
+ * interactions. This is not defined in the R4 specification but has been
+ * proposed for R5. The IBM FHIR Server implements it for R4 because it is 
+ * useful to support system-system synchronization scenarios
+ * @formatter:off
+ * _sort=_lastUpdated            - ORDER BY last_updated ASC
+ * _sort=-_lastUpdated (default) - ORDER BY last_updated DESC
+ * _sort=none                    - ORDER BY resource_id ASC
+ * @formatter:on
```

Comment:
the formatter tags will keep eclipse from squashing these lines, but javadoc / html will still lose the linebreaks unless you do something like this:
```suggestion
 * <pre>
 * _sort=_lastUpdated            - ORDER BY last_updated ASC
 * _sort=-_lastUpdated (default) - ORDER BY last_updated DESC
 * _sort=none                    - ORDER BY resource_id ASC
 * </pre>
```

Reply:
ok